docs/src/pages/options/animations.md

```diff
@@ -37,6 +37,10 @@ If you are building a website, you can also skip configuring quasar.conf.js and
 </head>
 ```
 
+::: warning
+It should be noted that when you import Animate.css through the `<link>` tag, all animation CSS classes must be prefixed with `animate__`. This is a breaking change in the migration of Animate.css from v3 to v4. If you want to avoid using prefixes, you can import [compact version](https://animate.style/#migration).
```

Comment:
Change `you can import [compact version]` to `you can import the [compact version]`

Reply:
Also, make a note that if using Quasar, then no additional changes are needed.

Reply:
OK, the modification is complete, thank you 😀